src/options/view.rs

```diff
@@ -1,19 +1,22 @@
-use output::{View, Mode, grid, details};
-use output::grid_details::{self, RowThreshold};
-use output::table::{TimeTypes, Environment, SizeFormat, Columns, Options as TableOptions};
-use output::time::TimeFormat;
+use lazy_static::{lazy_static, __lazy_static_create, __lazy_static_internal};
```

Comment:
I feel you should use `use lazy_static::*;`, because `__lazy_static_create` and `__lazy_static_internal` are implementations details of lazy_static.